docs/en/api/promql-service.md

```diff
@@ -0,0 +1,632 @@
+
+| Query Labels                     | Scope           | Expression Example                                                                             |
+|----------------------------------|-----------------|------------------------------------------------------------------------------------------------|
+| layer, service                   | Service         | service_cpm{service='$service', layer='$layer'}                                                |
+| layer, service, service_instance | ServiceInstance | service_instance_cpm{service='$service', service_instance='$service_instance', layer='$layer'} |
+| layer, service, endpoint         | Endpoint        | endpoint_cpm{service='$service', endpoint='$endpoint', layer='$layer'}                         |
+
+
+
+### Common Value Metrics
+- Query Labels: 
+```shell
+{General labels}
+```
+
+- Expression Example: 
+```shell
+service_cpm{service='agent::songs', layer='GENERAL'}
+```
+
+- Result (Instant Query):
+```json
+{
+    "status": "success",
+    "data": {
+        "resultType": "vector",
+        "result": [
+            {
+                "metric": {
+                    "__name__": "service_cpm",
+                    "layer": "GENERAL",
+                    "scope": "Service",
+                    "service": "agent::songs"
+                },
+                "value": [
+                    1677490740,
+                    "3"
+                ]
+            }
+        ]
+    }
+}
+```
+
+
+### Labeled Value Metrics
+- Query Labels:
+```shell
+--{General labels}
+--labels: Used to filter the value labels to be returned
+--relabels: Used to rename the returned value labels
+note: The number and order of labels must match the number and order of relabels.
+```
+
+- Expression Example:
+```shell
+service_percentile{service='agent::songs', layer='GENERAL', labels='0,1,2', relabels='P50,P75,P90'}
+```
+
+- Result (Instant Query):
+```json
+{
+  "status": "success",
+  "data": {
+    "resultType": "vector",
+    "result": [
+      {
+        "metric": {
+          "__name__": "service_percentile",
+          "label": "P50",
+          "layer": "GENERAL",
+          "scope": "Service",
+          "service": "agent::songs"
+        },
+        "value": [
+          1677493380,
+          "0"
+        ]
+      },
+      {
+        "metric": {
+          "__name__": "service_percentile",
+          "label": "P75",
+          "layer": "GENERAL",
+          "scope": "Service",
+          "service": "agent::songs"
+        },
+        "value": [
+          1677493380,
+          "0"
+        ]
+      },
+      {
+        "metric": {
+          "__name__": "service_percentile",
+          "label": "P90",
+          "layer": "GENERAL",
+          "scope": "Service",
+          "service": "agent::songs"
+        },
+        "value": [
+          1677493380,
+          "0"
+        ]
+      }
+    ]
+  }
+}
+```
+
+### Sort Metrics
+- Query Labels:
+```shell
+--parent_service: Name of the parent service
```

Comment:
This should be optional? Do we support that? Such as we have a TopN slow service list.
I know we are not recommending that due to Layers mixed, but in theory, we could do this query if users have only one layer, right?